app/controllers/communities_controller.rb

```diff
@@ -41,26 +41,31 @@ def create
 
     if @community.save
       Subscription.create!(community_id: @community.id, account_id: current_account.id)
+      flash[:notice] = "Community Created Successfully "
       redirect_to communities_path
     else
+      flash[:alert] = "Please fill all required Fields"
       render :new
     end
   end
 
   def update
     if @community.update(community_values)
+      flash[:notice] = "Community Updated Successfully"
```

Comment:
add all the success and error messages to local.yml file and use rails localize for that